labels-owners.yaml

```diff
@@ -0,0 +1,157 @@
+
+# This file provides information about labels and owners behind them
+#
+# Please keep this file alphabetized by label name and owner name
+
+repositories:
+    kubeflow:
+        area/airgapped:
+          owners:
+            - krishnadurai
+            - swiftdiaries
+        area/applications:
+          owners:
+            - kkasravi
+        area/centraldashboard:
+          owners:
+            - avdaredevil
+            - prodonjs
+        area/docs:
+          owners:
+            - sarahmaddox
+        area/example:
+          owners:
+            - dansanche
+            - jinchihe
+            - lluunn
+            - texasmichelle
+        area/gcp:
+          owners:
+            - gabrielwen
+            - jlewi
+            - kunmingg
+            - lluunn
+            - richardsliu
+            - zhenghuiwang
+        area/inference:
+          owners:
+            - animeshsingh
+            - cliveseldon
+            - ellis-bigelow
+            - rakelkar
+            - yuzisun
+        area/istio:
+          owners:
+            - krishnadurai
+            - kunmingg
+            - lluunn
+        area/jupyter:
+          owners:
+            - avdaredevil
+            - ioandr
+            - kimwnasptd
+            - pdmack
+            - prodonjs
+            - vkoukis
+        area/kubebench:
+          owners:
+            - andreyvelich
+            - swiftdiaries
+            - xyhuang
+        area/kustomize:
+          owners:
+            - jinchihe
+            - kkasravi
+            - swiftdiaries
+        area/logging:
+          owners:
+            - jlewi
+            - kunmingg
+        area/mxnet:
+          owners:
+            - gaocegege
+            - suleisl2000
+            - wackxu
+        area/openmpi:
+          owners:
+            - alsrgv
+            - jiezhang
+            - jlewi
+        area/openvino:
+          owners:
+            - kkasravi
+        area/pipelines:
+          owners:
+            - ark-kun
+            - ironPan
+            - neuromage
+            - paveldournov
+        area/sdk:
+          owners:
+            - animeshsinghj
+            - jlewi
+            - jinchihe
+        area/seldon:
+          owners:
+            - cliveseldon
+        area/testing:
+          owners:
+            - gabrielwen
+            - jlewi
+            - kunmingg
+            - lluunn
+            - richardsliu
+            - zhenghuiwang
+        area/tfjob:
+          owners:
+            - gaocegege
+            - johnugeorge
+            - ichardsliu
```

Comment:
Richard's github id is wrong.

Reply:
Remove @gaocegege its mostly  @richardsliu

Reply:
I can help maintain the tf-operator while I am more interested in these unmatured operators like xgboost, mxnet, mpi, and others.